src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -142,6 +143,25 @@ internal class OktaApi : IOktaApi {
 		throw new BmxException( "Account could not be found" );
 	}
 
+	async Task<string> IOktaApi.GetMeResponseAsync( string sessionId ) {
+
+		try {
+			using var meResponse = await _httpClient.GetAsync( "users/me" );
+
+			if( !meResponse.IsSuccessStatusCode ) {
+				return "";
+			}
+
+			var meJson = await meResponse.Content.ReadAsStringAsync();
+			var me = JsonSerializer.Deserialize( meJson, SourceGenerationContext.Default.OktaMeResponse )!;
```

Comment:
`ReadFromJsonAsync` exists:
https://learn.microsoft.com/en-us/dotnet/api/system.net.http.json.httpcontentjsonextensions.readfromjsonasync?view=net-7.0

Reply:
also, not a big concern, but it's theoretically possible for the deserialization to return null. so not the ideal to suppress the null warning.